dmutils/documents.py

```diff
@@ -49,10 +49,11 @@ def validate_documents(files):
     return errors
 
 
-def upload_document(uploader, documents_url, service, field, file_contents, public=True):
+def upload_document(uploader, bucket_name, documents_url, service, field, file_contents, public=True):
```

Comment:
There is an attribute of the uploader called bucket name, this is the short name. I guess it could be a property. Actually, that would work really nicely.